.github/actions/depot_tools/action.yaml

```diff
@@ -8,11 +8,13 @@ runs:
   using: "composite"
   steps:
       - name: Get depot tools
-        env:
-          DEPOT_TOOLS_REVISION: 22e5a04e5975a4308c15b45b34b1b120bd0c7224
```

Comment:
Maybe mention in the PR description
this undoes #5200?

Reply:
Done